src/pages/Designer/Designer.js

```diff
@@ -25,35 +25,111 @@ const Designer = () => (
   <Box>
     <AppHeader />
     <Main>
-      <Box margin={{ top: 'xlarge' }}>
-        <Stack>
-          <Box>
-            <Box alignSelf="center">
-              <Text
-                margin={{
-                  left: 'large',
-                }}
-              >
-                Grommet Tools
-              </Text>
-              <Heading
-                size="xlarge"
-                margin={{
-                  top: 'none',
-                  bottom: 'medium',
-                  left: 'large',
-                }}
+      <ResponsiveContext.Consumer>
+        {(responsive) =>
+          responsive === 'large' || responsive === 'xlarge' ? (
```

Comment:
Instead of having a super long 'if' statement, try to simplify the code using functional components.
Something like:
xyz ? <Component1 /> : <Component2 />.